environments/production/terraform.yml

```diff
@@ -299,6 +299,19 @@ servers:
     count: 10
     server_auto_recovery: true
 
+  - server_name: "formplayer_a2{i}-production"
+    server_instance_type: r6a.2xlarge
+    network_tier: "app-private"
+    az: "a"
+    volume_size: 50
```

Comment:
I assume the volume size doesn't need to be as large as existing machines since this will distribute users across twice the number of machines? Just curious how you settled on 50.

Reply:
Yeah that was my logic. Just save a bit of money (but also really not very much money now that I think of it). But yeah, I expect disk usage to approximately halve when we double the number of machines.